supervisor/homeassistant/module.py

```diff
@@ -393,63 +393,60 @@ async def end_backup(self) -> None:
     async def backup(
         self, tar_file: tarfile.TarFile, exclude_database: bool = False
     ) -> None:
-        """Backup Home Assistant Core config/ directory."""
-        await self.begin_backup()
-        try:
+        """Backup Home Assistant Core config/directory."""
+
```

Comment:
```suggestion
```
I guess I don't care too much but we used to have a linter that complained every time there was a newline in between the docstring and the start of the code, not sure what happened to that.